lib/messenger.rb

```diff
@@ -0,0 +1,30 @@
+module Messenger
+
+  def send_sms(number, content)
+    sid = ENV["TWILIO_ACCT_SID"]
+    auth = ENV["TWILIO_AUTH"]
+
+    @client = Twilio::REST::Client.new sid, auth
+
+    from = "+14123854063"
+
+    # The following try-rescue block is needed in case user unsubscribe
+    # if the user ubsubscribe and we attempt to message them
+    # the api will report an error
+
+    begin
+      message = @client.account.messages.create(
+          :from => from,
+          :to => '+1'+number,
+          :body => content
+      )
+    rescue Twilio::REST::RequestError => e
+      case e.code
+        when 21610
```

Comment:
I think you're missing my point. If this error means that a user un-subscribed from Twilio, we should probably make note of that locally so that we don't try to message that user again in the future, no?